django/forms/widgets.py

```diff
@@ -845,9 +845,7 @@ def get_context(self, name, value, attrs):
         return context
 
     def id_for_label(self, id_):
-        if id_:
-            id_ += '_0'
-        return id_
+        return '%s_0' % id_
```

Comment:
This hooks should return an ID value that corresponds to the first ID in the widget's tags (see [docs](https://docs.djangoproject.com/en/3.0/ref/forms/widgets/#django.forms.Widget.id_for_label)), so [if ID is empty](https://github.com/django/django/pull/12672/files#diff-aee6730bae795d31efec3c5d014804a9R838-R842) this should also return an empty value i.e.
```python
>>> id_for_label(None)
None
>>> id_for_label('')
''
```
not
```python
>>> id_for_label(None)
'None_0
>>> id_for_label('')
'_0'
```
```